provider/lxd/environ.go

```diff
@@ -133,8 +135,12 @@ func (env *environ) initProfile() error {
 	return err
 }
 
+func (env *environ) shortModelUUID() string {
+	return env.uuid[:shortModelIdLength]
+}
+
 func (env *environ) profileName() string {
```

Comment:
discussion:
hi adis, just curious about a detail regarding this implementation. Destroy() seems to be used in the client side as well in the kill controller command when we kill a controller. If users were to upgrade their client and does a kill-controller command on old controller, the profile wouldn't be deleted right? Not sure in this case do we have to handle possible users' expectations that the profile would be deleted in kill-controller once the client is upgraded? Perhaps ian could weigh in on this as well.